package/network/services/dropbear/Makefile

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2006-2016 OpenWrt.org
+# Copyright (C) 2006-2019 OpenWrt.org
```

Comment:
Don't update copyright unless you're affiliated with OpenWrt

Reply:
Got it. :)